src/content/docs/style-guide/writing-strategies/guidelines-documenting-ui.mdx

```diff
@@ -0,0 +1,121 @@
+
+For another example: if a new UI component helps customers with a specific procedure or use case, we may decide to only mention that UI component in the context of a specific procedure or tutorial, as opposed to documenting it in a dedicated UI reference doc. 
+
+Usually, there **will** be something we think is worth documenting about a new UI component, but not always. Having a default of "no documentation" helps us formalize our goal of concise and efficient documentation. 
+
+## Avoid documenting the obvious or intuitive [#avoid-documenting-obvious]
+
+If new UI includes features that are common in many UI experiences and therefore rather intuitive, you should avoid documenting those. And, relatedly, you should try to document what is **not** obvious about using a UI experience. Taking this approach will help you arrive at the most succinct and helpful documentation. 
+
+For example, many UI experiences for many applications include CRUD functionality (create, read, update, delete). Much of this functionality will be straightforward to anyone who has used any software, and this means we can avoid belaboring these more intuitive aspects of our product (and focus on the less obvious and more helpful aspects). For an example of avoiding documenting obvious UI functionality, see [Example of minimal docs: Lookup table](#example-docs-lookup-table).
+
+## Avoid documenting New Relic features that are documented elsewhere [#avoid-documenting-general-platform-features] 
+
+We should avoid creating redundant documentation, as a general rule, because having redundant docs can make maintenance and updates much harder. 
+
+If we find ourselves writing about a feature that's available in other New Relic UI experiences and/or documented elsewhere in the docs, we should either not mention it or, if we think it's helpful to mention it, link to the existing docs for more detail. 
+
+If we notice a general platform feature that we don't currently document that we think should be, we should try to add that somewhere in the `Learn about the platform` section of docs (as opposed to shoehorning it in a fairly unrelated doc).
+
+Some small amount of redundancy can be okay, if we think it's helpful and if it's very short. For example, we sometimes mention specific attributes or config options that are helpful for specific contexts in more than one doc. But for anything above a couple sentences, we should try to avoid redundancy. 
+
+## Examples of documentation approaches [#example-docs]
+
+Here are some examples of approaches taken for different types of UI components and pages. Our featuring these examples doesn't mean that we think these docs are perfect, or must be emulated. We include them to give you some ideas for how to approach different types of projects. 
+
+### Example of minimal docs: Sign-in using Google [#example-doc-google]
```

Comment:
Maybe these can be in collapsers? this doc is very text heavy and hard to parse. i think collapsers can break up the page